blockprotocol/blockprotocol
# Delist invoices for users with incomplete Stripe subscriptions

## 🌟 What is the purpose of this PR?




This PR updates the payment history section of the billing page to not list past/upcoming invoices for users with an incomplete stripe subscription.

## 🔗 Related links





- [Asana task](https://app.asana.com/0/1203623179643287/1204059903170051/f) _(internal)_

## 🔍 What does this change?




See commits.

## 🛡 What tests cover this?



Manual testing.

## ❓ How to test this?



1. Checkout the branch and setup the local dev environment
2. Run the internal api locally
3. Login and got to the upgrade page without paying for a subscription (this will give the logged in user an `incomplete` subscription status)
4. Check that the payment history section on the billing page remains empty

## 📹 Demo




https://user-images.githubusercontent.com/42802102/221918391-9a99c58a-f378-4e3c-adfc-29faa64c6724.mov
